test/metabase/api/search_test.clj

```diff
@@ -812,23 +812,96 @@
         (is (= 11 (call-count)))))))
 
 (deftest snowplow-new-search-query-event-test
-  (testing "Send a snowplow event when a new global search query is made"
-    (snowplow-test/with-fake-snowplow-collector
-      (mt/user-http-request :crowberto :get 200 "search?q=test")
-      (is (=? {:data {"event"                "new_search_query"
-                      "runtime_milliseconds" pos?}
+  (testing "Send a snowplow event when a search query is triggered and context is passed"
+           (snowplow-test/with-fake-snowplow-collector
```

Comment:
Something's wrong with your calva auto-indentation configuration. `testing` blocks should have two space indentation. 
```suggestion
  (testing "Send a snowplow event when a search query is triggered and context is passed"
    (snowplow-test/with-fake-snowplow-collector
```
Either ask a BE eng to clean this up (I can do it) or huddle with someone to fix it

Reply:
I _think_ I solved it - calva seemed to figure out how to actually format this. But I'll make sure with you